Makefile

```diff
@@ -58,7 +58,7 @@ NO_VALUE ?= no_value
 #		VARIABLES
 ###############################
 PWD ?=  $(shell pwd)
-VERSION ?= $(shell helm show chart chart/k8gb/|awk '/appVersion:/ {print $$2}')
+VERSION ?= $(shell helm show chart chart/k8gb/ 2> /dev/null | awk '/appVersion:/ {print $$2}')
```

Comment:
What if `stderr` contains a valid error? it will be invisible right?

Reply:
@ytsarev I've tried to minimize the silencing impact only to this command, but yes, errors will be suppressed. 
Regarding possible valid errors for this command, I see issues like helm not installed, chart code missing, appVersion missing in the chart.

> > @somaritane which version of helm do you use?
> > Mine looks silent
> 
> @ytsarev mine is v3.5.4 I can downgrade and check.

v3.3.4 version acts the same for me

Reply:
```suggestion
VERSION ?= $(shell helm show chart chart/k8gb/ 2>&1 |grep -v 'found symbolic link in path' | awk '/appVersion:/ {print $$2}')
```

Reply:
@ytsarev , since awk is "interested" in `awkVersion:` only, it will suppress the rest of stdout, including the redirected stderr with possible errors.